### DISCUSSION TITLE
Why "fediverse:creator" instead of "fediverse:author"?

### DISCUSSION THREAD

**Discussion by antimundo** [mastodon/mastodon#32328] [General]
[Mastodon 4.3](https://blog.joinmastodon.org/2024/10/mastodon-4.3/) introduced a new meta tag that you can add to the HTML of your website, so publishers can feature the fediverse profile of the page author in the link previews on Mastodon. [This post shows how it works](https://rknight.me/blog/setting-up-mastodon-author-tags/).

```
<meta name="fediverse:creator" content="@antimundo@mastodon.gamedev.place" />
```

And this left me wondering, why is this tag "fediverse:creator" instead of "fediverse:author"? There is already a standard meta tag called "author", you can see [this is a HTML standard](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta/name) in the the  [HTML specification](https://html.spec.whatwg.org/multipage/semantics.html#meta-author).

So if we already have an accepted standard of using the keyword "author" why not just keep using it?

```
<meta name="author" content="Antimundo">
<meta name="fediverse:author" content="@antimundo@mastodon.gamedev.place" />
```

instead of the current implementation:

```
<meta name="author" content="Antimundo">
<meta name="fediverse:creator" content="@antimundo@mastodon.gamedev.place" />
```

**Comment by jartigag** [comment]
I think it's because of the ["twitter:creator" meta tag](https://developer.x.com/en/docs/x-for-websites/cards/overview/markup)

**Comment by trwnh** [comment]
there is also rel=author:

```html
<meta name="author" content="a (trwnh.com)" />
<link rel="author" href="https://trwnh.com" />
```

there is a proposal to use the activitystreams/activitypub type for this:

```html
<meta name="author" content="someone" />
<link rel="author" href="https://mastodon.example/@someone" type="application/activity+json" />
```

**Comment by andypiper** [comment]
This was discussed (at length) by the team before implementation, and during the early implementation and betas. The concept here was that it was similar to OpenGraph format (but, has no relation to the old twitter:creator tag, that's coincidental). You can see more [here](https://github.com/mastodon/mastodon/pull/30398#issuecomment-2127233433) (and elsewhere in the repository). We are looking at aligning with the ActivityStreams/ActivityPub type in the future and are following the W3C working group discussions.
